lldb/test/Shell/SymbolFile/DWARF/dwo-miss-getdie-ouside-cu-error.c

```diff
@@ -0,0 +1,29 @@
+/// Check that LLDB does not emit "GetDIE for DIE {{0x[0-9a-f]+}} is outside of its CU"
+/// error message when user is searching for a matching symbol from .debug_names
+/// and fail to locate the corresponding .dwo file.
+
+/// -gsplit-dwarf is supported only on Linux.
```

Comment:
Might be wrong, but isn't it also supported on Windows?

Reply:
I copied this from dwo-missing-error.test:
https://github.com/llvm/llvm-project/blob/main/lldb/test/Shell/SymbolFile/DWARF/dwo-missing-error.test#L5-L6

Since I do not have a windows machine to test so feels safer to limit for Linux. Let me know if you find otherwise and I can update the comment.